src/licensedcode/data/licenses/embedthis-extension.LICENSE

```diff
@@ -6,14 +6,15 @@ category: Free Restricted
 owner: Embedthis
 homepage_url: https://www.embedthis.com/licensing/extension.html
 spdx_license_key: LicenseRef-scancode-embedthis-extension
+minimum_coverage: 95
 ---
 
-Embedthis Extension License
+{{Embedthis Extension License}}
```

Comment:
We should not have key phrases in license texts. `minimum_coverage: 95` should be enough... The curly braces are making this hard to use as a reference text, which is the primary goal